aptos-node/src/lib.rs

```diff
@@ -253,46 +255,22 @@ where
     let aptos_root_key_path = test_dir.join("mint.key");
 
     // If there's already a config, use it. Otherwise create a test one.
-    let config = if validator_config_path.exists() {
+    let config = if let Some(config) = config {
```

Comment:
nit: This looks odd. Why not move it out to the call site? 😄

Reply:
Because this function does a bunch of magic overrides on the config for local testing. A larger refactor is probably due, eg separating all that into its own function.